.golangci.yml

```diff
@@ -43,6 +43,9 @@ linters:
     - unused
     - varcheck
     - godot
+    - unparam
+    - unused
+    - varcheck
```

Comment:
These two were already in the list, a couple of entries above.

Reply:
🙈 I messed up something while rebasing, will fix it.